packages/react-router-website/examples/PreventingUnload.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import Router from 'react-router-dom/BrowserRouter'
+import Route from 'react-router-dom/Route'
+import Link from 'react-router-dom/Link'
+import BrowserPrompt from '../../react-router-dom/BrowserPrompt'
```

Comment:
Please update the example to import everything from `react-router-dom` (see the other examples which have recently been updated.